cmake/findBLIS.cmake

```diff
@@ -39,6 +39,9 @@ elseif(EXISTS      "${BUILD_DIR}/deps/blis/lib/libblis.a")
 elseif(EXISTS      "/usr/local/lib/libblis.a")
     set( BLIS_LIB /usr/local/lib/libblis.a )
     set( BLIS_INCLUDE_DIR /usr/local/include/blis )
+elseif(EXISTS "${CMAKE_CURRENT_BINARY_DIR}/../deps/blis")
```

Comment:
Does Line-36 (elseif(EXISTS "${BUILD_DIR}/deps/blis/lib/libblis.a")) not do this?

Reply:
It doesn't. Where is BUILD_DIR set?

Reply:
If the `${BUILD_DIR}` is not set then we could replace all `${BUILD_DIR}`  with `${CMAKE_CURRENT_BINARY_DIR}/..` that should fix the issue as well.